data/data/powervs/main.tf

```diff
@@ -4,21 +4,60 @@ provider "ibm" {
   zone             = var.ibmcloud_zone
 }
 
+data "ibm_resource_group" "cos_group" {
+  name = var.powervs_resource_group_name
+}
+
+resource "ibm_resource_instance" "cos_instance" {
+  name              = var.powervs_cos_instance_name
+  resource_group_id = data.ibm_resource_group.cos_group.id
+  service           = "cloud-object-storage"
+  plan              = "standard"
+  location          = var.powervs_cos_instance_location
+}
+
+# Create an IBM COS Bucket to store ignition
+resource "ibm_cos_bucket" "ignition" {
+  bucket_name          = "${var.powervs_cluster_id}-ignition"
```

Comment:
it will be good if you can make it more explicit like `*-bootstrap-ign`

Reply:
I used *-ignition here because (i believe) the master node can share this bucket to serve it's master.ign file

Reply:
nope, master and worker nodes get ignition directly from the MCO(machine config operator)

Reply:
Ahh fair enough, fixed!